libmambapy/src/main.cpp

```diff
@@ -543,6 +543,11 @@ PYBIND11_MODULE(bindings, m)
         .def_readwrite("custom_channels", &Context::custom_channels)
         .def_readwrite("custom_multichannels", &Context::custom_multichannels)
         .def_readwrite("default_channels", &Context::default_channels)
+        .def_property(
+            "conda_build_local_paths",
```

Comment:
I'm using `conda_build_local_paths` because conda has a read only property with the same name.